include/tvm/relay/op_attr_types.h

```diff
@@ -72,7 +72,8 @@ using FTVMCompute = runtime::TypedPackedFunc<
  * \return schedule The computation schedule.
  */
 using FTVMSchedule = runtime::TypedPackedFunc<
-  Schedule(const Array<Tensor>& outs,
+  Schedule(const Attrs& attrs,
```

Comment:
this change doesn't belong to this PR

Reply:
Thanks @vinx13 I have cleaned up the changes